### PR TITLE
Reformat all HSE C source files with new clang format

### PR DESCRIPTION
**Pull request by alexttx**
This PR/branch will be rebased to origin/master after pr #592 is merged.

### CODE REVIEW

File: cli/include/hse/cli/rest/client.h [hunk 1]
```diff
@@ -23,8 +26,7 @@ rest_client_init(const char *socket_path);
  * hsettp tool.
  */
 merr_t
-rest_client_fetch(
-    const char *method,
+rest_client_fetch(const char *method,
```
Comment (tristan957) [inline cli/include/hse/cli/rest/client.h:29]: I don't understand what is going on here?

Comment (alexttx) [reply cli/include/hse/cli/rest/client.h:29]: I don't know exactly what combination of settings caused this, but it's the lesser of 25 evils IMO.

Comment (tristan957) [reply cli/include/hse/cli/rest/client.h:29]: It was way better before. This is like the most evil of all 25 evils :).

Comment (beckerg) [reply cli/include/hse/cli/rest/client.h:29]: soon there will be nothing left of my tongue..

Comment (tristan957) [reply cli/include/hse/cli/rest/client.h:29]: I will investigate in a bit.

**Review by tristan957 — APPROVED** [review]

**Comment by alexttx** [comment]
@beckerg need your approval.

**Review by beckerg — APPROVED** [review]

**Pull request merged**